### PR TITLE
#150 If networkMode host ForListeningPort does not work

### PR DESCRIPTION
**Pull request by Bablzz**
Hi everybody!

Maybe It is not an ideal solution.

### CODE REVIEW

**Comment by ClaytonNorthey92** [comment]
could we add tests for [Endpoint](https://github.com/testcontainers/testcontainers-go/blob/master/docker.go#L57) and [PortEndpoint](https://github.com/testcontainers/testcontainers-go/blob/master/docker.go#L75) that behave with `networkMode="host"`?

**Comment by Bablzz** [comment]
I think I can

**Comment by ClaytonNorthey92** [comment]
@Bablzz what you added looks good 👍 thank you!

This looks good to me, I don't use `networkMode="host"` too often, so I am not sure of all of the side effects besides the port mappings.  But I think the test coverage is good, and the code is written well so that this change is very minimally invasive.

File: docker_test.go [hunk 1]
```diff
@@ -136,6 +136,97 @@ func TestContainerWithNetworkModeAndNetworkTogether(t *testing.T) {
...
+
+func TestContainerWithHostNetworkAndPortEndpoint(t *testing.T) {
+	nginxPort := "80/tcp"
+	ctx := context.Background()
+	gcr := GenericContainerRequest{ContainerRequest: ContainerRequest{
+		Image:       "nginx",
+		SkipReaper:  true,
+		NetworkMode: "host",
+		WaitingFor:  wait.ForListeningPort(nat.Port(nginxPort)),
+	},
```
Comment (mdelapenya) [inline docker_test.go]: I guess this format is not correct, could you fix that? 🙏

Comment (Bablzz) [reply docker_test.go]: Only format?! It's not a problem to fix it. :ok_hand: 
This a strange that **go fmt** does not reformat a code.

**Comment by Bablzz** [comment]
@ClaytonNorthey92 thanks a lot! I hope tests help us :smile:

**Comment by gianarb** [comment]
Thanks a lot @Bablzz !

**Comment by Bablzz** [comment]
you are welcome @gianarb !

**Pull request merged**